src/Skoruba.IdentityServer4.STS.Identity/Helpers/StartupHelpers.cs

```diff
@@ -309,6 +312,26 @@ private static RegisterConfiguration GetRegistrationConfiguration(IConfiguration
         {
             var externalProviderConfiguration = configuration.GetSection(nameof(ExternalProvidersConfiguration)).Get<ExternalProvidersConfiguration>();
 
+            if (externalProviderConfiguration.UseSaml2Provider)
+            {
+                authenticationBuilder.AddSaml2(options => 
+                {
+                    options.SignInScheme = IdentityConstants.ExternalScheme;
```

Comment:
I'm not sure about the implications of this. However, if we would use IdentityServer's default external scheme, AccountController.ExternalLoginCallback fails because `_signInManager.GetExternalLoginInfoAsync` returns null.